### DISCUSSION TITLE
whichwrap in chadrc.lua - settings overrided by someone

### DISCUSSION THREAD

**Discussion by akhasano** [NvChad/NvChad#3021] [Q&A] [answered]
Hello,

I hate when h,l switched to another line at the begin or at the end string.
I am trying to configure:
vim.cmd('set whichwrap=b,s') 

but when i request
:set ww?
i am getting 
whichwrap=<>[]hl,b,s

who is overriding settings?
Thanks

**Comment by siduck** [comment, marked as answer]
please put the whichwrap in your options file which loads `nvchad.options` module, so it will override our default option
